### DISCUSSION TITLE
Button for launching scripts

### DISCUSSION THREAD

**Discussion by FillyHolly** [nasa/openmct#3805] [Community Q&A]
Hello! Is there a button to stop / start telemetry? Is it possible to add a button function to run separate scripts that can be created in DisplayLayout?

**Comment by DusterTheFirst** [comment]
This is not the place to ask for personal help, the issues page is meant for issues with the project that the maintainers need to address. 

Answering your question, there are a lot of tools provided by openmct to control your telemetry. You can make custom components for the buttons, or you can setup an indicator button to control it. If you have not, you should read through the [Tutorial](https://github.com/nasa/openmct-tutorial/blob/HEAD/README.md) or the [API documentation](https://github.com/nasa/openmct/blob/master/API.md#indicators)

**Comment by DusterTheFirst** [comment]
This may be better suited as a discussion topic: https://github.com/nasa/openmct/discussions/categories/community-q-a

**Comment by jvigliotta** [comment]
Thank you @DusterTheFirst and yes this is a question better suited to our Discussions. @FillyHolly I'll actually move the issue over there as a discussion topic.

**Comment by jvigliotta** [comment]
@FillyHolly also, our TelemetryTables plugin (and I believe our Plots Plugins) have this ability, though it doesn't really pause telemetry so much as cache incoming while the updates to the view are "paused". When unpaused, cached data is processed and displayed for the period it was paused.
